plugins/v1/repository_gcs/plugin.py

```diff
@@ -0,0 +1,89 @@
+# Licensed to Elasticsearch B.V. under one or more contributor
+# license agreements. See the NOTICE file distributed with
+# this work for additional information regarding copyright
+# ownership. Elasticsearch B.V. licenses this file to you under
+# the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#	http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import logging
+import os
+
+from esrally.utils import process
+from esrally import exceptions
+
+LOGGER_NAME = "rally.provisioner.repository_gcs"
```

Comment:
Nit: Should be `esrally...`, not `rally...`.

Reply:
I picked this up from the 6.2 branch in https://github.com/elastic/rally-teams/blob/6.2/plugins/v1/x_pack/plugin.py#L25, I guess we should rename that too in another PR?

Reply:
Oh, I see. I am not sure it's worth the effort given that 6.2 is already end of life?